jena/src/main/java/com/inrupt/client/jena/JenaBodyHandlers.java

```diff
@@ -43,14 +47,28 @@
 public final class JenaBodyHandlers {
 
     private static final String CONTENT_TYPE = "Content-Type";
+    private static JsonService jsonService;
```

Comment:
Something feels wrong about the Jena module developing a dependency (even if weak) on the JSON module.

I follow why this is needed, but it feels off regardless.

Reply:
With the change proposed in 2ed0a03ba49d8c5fac59e0f01ae37ee3e545798c, this dependency can be removed. I thought that approach would create weird dependency cycles, but it appears to be working fine.